dispatcherd/service/process.py

```diff
@@ -75,7 +78,7 @@ class ProcessManager:
 
     def __init__(self, settings: LazySettings = global_settings) -> None:
         self.ctx = multiprocessing.get_context(self.mp_context)
-        self.finished_queue: multiprocessing.Queue = self.ctx.Queue()
+        self._finished_queue: Optional[multiprocessing.Queue] = self.ctx.Queue()
```

Comment:
The reason for adding `Optional` isn't clear to me. Can `self.ctx.Queue()` return `None` ?